### PR TITLE
add the example segment

### PR DESCRIPTION
**Pull request by Explorerhpx**
* add the example part
* update the acknowledgments

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -188,3 +241,47 @@ <h2>Acknowledgments</h2>
...
+
+  // function main() {
+  //   if (document.readyState === "loading") {
+  //     document.addEventListener("DOMContentLoaded", changeEvidenceColor);
+  //   } else {
+  //     changeEvidenceColor();
+  //   }
+  // }
+
+  // main();
```
Comment (bryant1410) [inline index.html]: Should we remove this?

File: index.html [hunk 2]
```diff
@@ -132,16 +132,76 @@ <h2>Abstract</h2>
...
 
-      <p id="affiliation">
-        <a href="https://umich.edu/">
-          <img id="um-vertical" alt="University of Michigan" src="img/um-vertical.png">
-        </a>
-      </p>
-
+  <section>
+    <div class="content">
+      <h2>Example</h2>
```
Comment (bryant1410) [inline index.html]: What about the following?

```suggestion
      <h2>Example from our Dataset</h2>
```

To give more context to someone who enters our website and doesn't know what it's about.

**Review by bryant1410 — APPROVED** [review]
It looks excellent!! Thanks a lot, Frank!!

File: index.html (hunk 1 above)
Comment (Explorerhpx) [reply index.html]: Sure! we can remove this.

File: index.html (hunk 2 above)
Comment (Explorerhpx) [reply index.html:140]: That makes sense to me.

**Pull request merged**